buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -415,4 +425,16 @@ class VagrantTestPlugin implements Plugin<Project> {
             }
         }
     }
+
+    private static TaskExecutionAdapter createReproListener(Project project) {
+        return new TaskExecutionAdapter() {
+            @Override
+            void afterExecute(Task task, TaskState state) {
+                final String gradlew = Os.isFamily(Os.FAMILY_WINDOWS) ? "gradlew" : "./gradlew"
+                if (state.failure != null) {
+                    println "REPRODUCE WITH: ${gradlew} ${task.path} -Dtests.seed=${project.testSeed} "
```

Comment:
Using `task.path` only works because this listener is always be added/removed in doFirst/doLast on a task. Rather that depend on that behavior, why not pass in the task name to `createReproListener`?

Reply:
Gotcha - my thinking was to prevent the repro message from mistakenly having a different task than the one that triggered the listener. But since the listener is added to the project, not the task, that makes it less clear what's happening. I'll have it pass the task as a parameter